Recruit-Training/Basic-Recruit-Training/person/main.cpp

```diff
@@ -19,15 +19,21 @@ void compareAnswers(const std::string myAnswer, const std::string correctAnswer)
 }
 // Swaps a & b.
 void mySwap(int& a, int& b)
-{
+{   
+    int temp;
+    temp=a;
+    a=b;
+    b=temp;
```

Comment:
Please leave a space between your operators. Like `a = b` and not `a=b`

Reply:
ohk i got that.
Shall I do it again??
I have configured my git, but I cant see github icon, I have sent you the ss of config on discord.

Reply:
I just updated this file